airflow/utils/serializers/bignum.py

```diff
@@ -0,0 +1,55 @@
+#
```

Comment:
We should have a doc about this too - how to add a custom serializer

Reply:
Yes